.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
```

Comment:
Would "Semantic Kernel Starters" be a better name given this repo contains .NET and Python projects and will include Java samples also in the future

Reply:
Fixed!